proxyfinder_test.go

```diff
@@ -72,3 +76,27 @@ func TestProxyWithoutPort(t *testing.T) {
 	pf := newProxyFinder(pacServer.URL, net.InterfaceAddrs)
 	checkProxyForURL(t, pf, "https://www.anz.com.au/", &url.URL{Host: "proxy.anz.com:80"})
 }
+
+func TestPacFromFilesystem(t *testing.T) {
+	//s := &proxyFinderTestServer{toAddrs("10.0.0.1"), "proxy.anz.com"}
+
+	content := []byte("function FindProxyForURL(url, host) { return \"PROXY proxy.example.com:80\"}")
+	tmpfile, err := ioutil.TempFile("", "test.pac")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer os.Remove(tmpfile.Name())
+
+	if _, err := tmpfile.Write(content); err != nil {
+		log.Fatal(err)
+	}
+	if err := tmpfile.Close(); err != nil {
+		log.Fatal(err)
+	}
```

Comment:
Again, you can use require.Nil here:

```
require.Nil(t, tmpfile.Write(content))
require.Nil(t, tmpfile.Close())
```

This should make the function concise enough that you can get rid of all the blank lines.